book/src/pong-tutorial/pong-tutorial-02.md

```diff
@@ -296,7 +353,51 @@ on the sprite sheet to be displayed.
 Then, using the `WithSpriteRender` trait, we can easily modify our
 entity creation code to have the entities display the sprite.
 
-```rust,ignore
+```rust,no_run,noplaypen
+# extern crate amethyst;
+# use amethyst::prelude::*;
+# use amethyst::assets::{Loader, AssetStorage};
+# use amethyst::renderer::{Texture, PngFormat, WithSpriteRender};
+# use amethyst::core::{Transform, GlobalTransform};
+# enum Side {
+#   Left,
+#   Right,
+# }
+# struct Paddle;
+# impl amethyst::ecs::Component for Paddle {
+#   type Storage = amethyst::ecs::VecStorage<Paddle>;    
+# }
+# impl Paddle {
+#   fn new(side: Side) -> Paddle {
+#       Paddle
+#   }
+# }
+# struct MyState;
+# impl<'a, 'b> State<GameData<'a, 'b>> for MyState {
+# fn on_start(&mut self, data: StateData<GameData>) {
+# let world = data.world;
+# let spritesheet = {
+#   let loader = world.read_resource::<Loader>();
+#   let texture_storage = world.read_resource::<AssetStorage<Texture>>();
+#   loader.load(
+#       "texture/pong_spritesheet.png",
+#       PngFormat,
+#       Default::default(),
+#       (),
+#       &texture_storage,
+#   )
+# };
+# use amethyst::renderer::Sprite;
+# const PADDLE_HEIGHT: f32 = 16.0;
+# const PADDLE_WIDTH: f32 = 4.0;
+# let sprite = Sprite {
+#   left: 0.0,
+#   right: PADDLE_WIDTH,
+#   top: 0.0,
+#   bottom: PADDLE_HEIGHT,
+# };
+# let left_transform = Transform::default();
+# let right_transform = Transform::default();
```

Comment:
the transform creation should be shown

Reply:
It is shown in the example above, I think repeating it would redundant.

Reply:
I must have missed it thanks